tools/vms-generator/utils/utils.go

```diff
@@ -38,31 +38,32 @@ import (
 )
 
 const (
-	VmiEphemeral         = "vmi-ephemeral"
-	VmiMigratable        = "vmi-migratable"
-	VmiFlavorSmall       = "vmi-flavor-small"
-	VmiSata              = "vmi-sata"
-	VmiFedora            = "vmi-fedora"
-	VmiSecureBoot        = "vmi-secureboot"
-	VmiAlpineEFI         = "vmi-alpine-efi"
-	VmiNoCloud           = "vmi-nocloud"
-	VmiPVC               = "vmi-pvc"
-	VmiBlockPVC          = "vmi-block-pvc"
-	VmiWindows           = "vmi-windows"
-	VmiKernelBoot        = "vmi-kernel-boot"
-	VmiSlirp             = "vmi-slirp"
-	VmiMasquerade        = "vmi-masquerade"
-	VmiSRIOV             = "vmi-sriov"
-	VmiWithHookSidecar   = "vmi-with-sidecar-hook"
-	VmiMultusPtp         = "vmi-multus-ptp"
-	VmiMultusMultipleNet = "vmi-multus-multiple-net"
-	VmiHostDisk          = "vmi-host-disk"
-	VmiGPU               = "vmi-gpu"
-	VmiARM               = "vmi-arm"
-	VmiMacvtap           = "vmi-macvtap"
-	VmTemplateFedora     = "vm-template-fedora"
-	VmTemplateRHEL7      = "vm-template-rhel7"
-	VmTemplateWindows    = "vm-template-windows2012r2"
+	VmiEphemeral              = "vmi-ephemeral"
+	VmiMigratable             = "vmi-migratable"
+	VmiFlavorSmall            = "vmi-flavor-small"
+	VmiSata                   = "vmi-sata"
+	VmiFedora                 = "vmi-fedora"
+	VmiSecureBoot             = "vmi-secureboot"
+	VmiAlpineEFI              = "vmi-alpine-efi"
+	VmiNoCloud                = "vmi-nocloud"
+	VmiPVC                    = "vmi-pvc"
+	VmiBlockPVC               = "vmi-block-pvc"
+	VmiWindows                = "vmi-windows"
+	VmiKernelBoot             = "vmi-kernel-boot"
+	VmiSlirp                  = "vmi-slirp"
+	VmiMasquerade             = "vmi-masquerade"
+	VmiSRIOV                  = "vmi-sriov"
+	VmiWithHookSidecar        = "vmi-with-sidecar-hook"
+	VmiWithVirtxmlHookSidecar = "vmi-with-virt-xml-sidecar-hook"
```

Comment:
Looking at how other consts are named, I think "XML" should be all uppercase letters.

And I know you're following the pre-existing example, but I think the name for this (as well as the file name for the example file) should be "vmi-with-virt-xml-hook-sidecar" to match the annotation name.